src/react/README.md

```diff
@@ -2,6 +2,31 @@
 
 This document provides a brief guide on how to use custom chat components and hooks provided by the Ably Chat SDK.
 
+## Installation
+
+To be able to use React for Chat, install the Chat SDK as normal:
+
+```sh
+npm install @ably/chat
+```
+
+For browsers, you can also include the React SDK directly into your HTML.
+
+```html
+<!-- Ably Chat also requires the core Ably SDK to be available -->
+<script src="https://cdn.ably.com/lib/ably.min-2.js"></script>
+<!-- You also need the base SDK -->
+<script src="https://cdn.ably.com/lib/ably-chat.umd.cjs-0.js"></script>
+<!-- Include React base library -->
+<script src="https://unpkg.com/react@18/umd/react.production.min.js"></script>
+<script src="https://cdn.ably.com/lib/ably-chat-react.umd.cjs-0.js"></script>
+<script>
+  const chatClient = new AblyChat.ChatClient(realtime);
```

Comment:
Should we include some example calling `AblyChatReact` here?

Reply:
In hindsight... I'm going to get rid of React on the CDN. It's very complicated for what is probably very little usage.